.ci/.e2e-platforms.yaml

```diff
@@ -0,0 +1,42 @@
+---
+PLATFORMS:
+  stack:
+    description: "Debian 10 AMD64, used specifically for running the stack"
+    image: "ami-0d90bed76900e679a"
+    instance_type: "c5.4xlarge"
+    username: admin
+  centos8_arm64:
+    description: "CentOS Stream 8 aarch64"
+    image: "ami-01cdc9e8306344fe0"
+    instance_type: "a1.large"
+    username: centos
+  centos8_amd64:
+    description: "CentOS Stream 8 x86_64"
+    image: "ami-045b0a05944af45c1"
+    instance_type: "c5.4xlarge"
+    username: centos
+  debian_arm64:
+    description: "Debian 10 ARM64"
+    image: "ami-06dac44ad759182bd"
+    instance_type: "a1.large"
+    username: admin
+  debian_amd64:
+    description: "Debian 10 AMD64"
+    image: "ami-0d90bed76900e679a"
+    instance_type: "c5.4xlarge"
+    username: admin
+  oracle_linux8:
+    description: "Oracle Linux 8 update 3 for x86_64 HVM"
+    image: "ami-00371eeb8fd8e0e16"
+    instance_type: "c5.2xlarge"
+    username: ec2-user
+  sles15:
+    description: "SUSE Linux Enterprise Server 15 SP3 (HVM), SSD Volume Type -  64bit ARM"
+    image: "ami-0f7cb53c916a75006"
+    instance_type: "c5.4xlarge"
+    username: ec2-user
+  fleet_elastic_pkg:
```

Comment:
I think we can remove this platform in a follow-up PR, as we can control the same with `provider: elastic-package` in the YAML file